keras/layers/containers.py

```diff
@@ -239,44 +246,59 @@ def add_input(self, name, input_shape, dtype='float'):
                                   'input_shape': input_shape,
                                   'dtype': dtype})
 
-    def add_node(self, layer, name, input=None, inputs=[],
-                 merge_mode='concat', concat_axis=-1, dot_axes=-1, create_output=False):
+    def add_node(self, layer, name, input=None, merge_mode='concat', concat_axis=-1, create_output=False):
```

Comment:
This is a change in API and will break some models. Will need docs and public announcements.